### ISSUE TITLE
s3fs fails build because of no matching fsspec

### ISSUE THREAD

**Issue by MordragT** [NixOS/nixpkgs#196321]
### Steps To Reproduce
1. Install dvc with aws integration

### Build log
```
ERROR: Could not find a version that satisfies the requirement fsspec==2022.8.2 (from s3fs) (from versions: none)
       > ERROR: No matching distribution found for fsspec==2022.8.2
```

### Additional context
I am running nixos on the unstable branch.

### Notify maintainers
@costrouc
@teh 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.68, NixOS, 22.11 (Raccoon), 22.11.20220926.62228cc`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Probably fsspec needs an update to 2022.8.2

**Comment by fabaff** [comment]
Will be fixed with #195701.

**Comment by MordragT** [comment]
Oh sorry for the noise, did not search the PRs.
